marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -173,8 +173,9 @@ class Jobs {
      }, this.config.JOBDATABASE));
   }
 
-  updateBatch(jobId, batchId, batchStatus, uris) {
+  updateBatch(jobId, batchId, batchStatus, uris, stackTrace) {
     let docObj = this.getBatchDoc(jobId, batchId);
+    let stackTraceObj = stackTrace[0];
```

Comment:
Need to check if an error exists. Also, you will need to adjust the flow.sjs code to pass the error when a batch has failed